starport-core/src/main/scala/com/krux/starport/db/tool/CreateUsers.scala

```diff
@@ -0,0 +1,85 @@
+package com.krux.starport.db.tool
+
+import com.krux.starport.config.StarportSettings
+import com.krux.starport.Logging
+import com.krux.starport.db.WaitForIt
+import com.krux.starport.db.table._
+import com.typesafe.config.ConfigValueFactory
+import scopt.OptionParser
+import slick.jdbc.PostgresProfile.api._
+import slick.jdbc.PostgresProfile.backend.DatabaseDef
+
+object CreateUsers extends App with Logging with WaitForIt with Schema {
+
+  lazy val starportSettings = StarportSettings()
+
+  case class Options(
+                      masterUser: String = "",
+                      masterPassword: String = "",
+                    ) {
+  }
+
+  def runQuery[T](query: DBIO[T], db: DatabaseDef): T = {
+    db.run(query).waitForResult
+  }
+
+  def createUser: DBIO[Int] = {
+    val userName = starportSettings.config.getString("krux.starport.jdbc.slick.properties.user")
+    val password = starportSettings.config.getString("krux.starport.jdbc.slick.properties.password")
+    logger.info(s"create role $userName")
+    sqlu"CREATE ROLE #$userName WITH LOGIN PASSWORD '#$password'"
+  }
+
+  def createReadOnlyUser: DBIO[Int] = {
```

Comment:
this is not required in running starport right now, and the config you specified below does not exist or used anywhere.

Reply:
The config has to be added for this to work. I thought @itsdawomb needed an RO user? If not I can drop this.

Reply:
For both RO and RW user? I can see a case for dropping the RO user, but the RW user should stay. It discourages the placement of the master user/pass in conf, and in general it is good practice to provide good bootstrap tools for an app (short of comprehensive database management of course.)

Reply:
yes, RW should stay